standalone_app/src/types/index.d.ts

```diff
@@ -1,6 +1,6 @@
 declare const IS_VSCODE: boolean
 
-type TrialValueNumber = number | "inf" | "-inf"
+type TrialValueNumber = number
 type TrialIntermediateValueNumber = number | "inf" | "-inf" | "nan"
```

Comment:
Do we also need to change here as well?
And `standalone_app/src/components/PlotIntermediateValues.tsx`?